gix-object/src/tree/mod.rs

```diff
@@ -249,14 +249,57 @@ impl Ord for EntryRef<'_> {
     }
 }
 
+/// TODO
+/// Keep in mind that the path separator always is `/`, independent of the platform.
```

Comment:
And on top of that, it's a relative path with only `Normal` path components. Also it's always represented as bunch of bytes.